src/Symfony/Component/VarDumper/Caster/StubCaster.php

```diff
@@ -89,4 +89,12 @@ public static function castScalar(ScalarStub $scalarStub, array $a, Stub $stub):
 
         return $a;
     }
+
+    public static function castVirtualStub(VirtualStub $virtualStub, array $a, Stub $stub): array
+    {
+        $stub->type = Stub::TYPE_SCALAR;
+        $stub->attr['value'] = $virtualStub->propertyType ?? '(no type)';
```

Comment:
a property with no type accepts `mixed`. Maybe that's what we should display.

Reply:
How do we distinguish it from a property that is actually typed `mixed` then?

Reply:
Er could display "mixed" or "mixed?" If we can distinguish it programmatically